Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -79,7 +78,8 @@ JS_DEFINE_ALLOCATOR(Window);
 void run_animation_frame_callbacks(DOM::Document& document, double now)
 {
     // FIXME: Bring this closer to the spec.
-    document.window()->animation_frame_callback_driver().run(now);
+    if (document.window())
+        document.window()->animation_frame_callback_driver().run(now);
```

Comment:
should we reach `run_animation_frame_callbacks` if document does not have a window?

Reply:
I've removed this check, I don't believe the error was ever triggered from this point. In order to get here the document must [be fully active](https://html.spec.whatwg.org/multipage/document-sequences.html#fully-active). After doing some digging, I believe that this implies that the document must also have an associated window object at this point.

Personally, I wouldn't mind a `VERIFY(document.window())` here, since it's not entirely obvious (to me at least) that the window object should definitely be non-null.